packages/kubernetes/data_stream/container_logs/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,32 @@
+---
+description: Pipeline for Kubernetes container logs
+processors:
+  - set:
+      field: ecs.version
+      value: '8.8.0'
+  - set:
+      field: service.name
+      value: '{{kubernetes.labels.app_kubernetes_io/name}}'
+      if: ctx?.kubernetes?.labels['app_kubernetes_io/name'] != null
```

This should be more efficient as Elasticsearch doesn't need to render a mustache template. It's also a bit more concise to use `ignore_empty_value` compared to the `if` condition.
```suggestion
  - set:
      field: service.name
      copy_from: kubernetes.labels.app_kubernetes_io/name
      ignore_empty_value: true
```

The same applies to the other fields as well.